chore: add Translations interface for language support oc: 5343

peco11 opened the pull request:
This commit adds the Translations interface to support multiple languages in the application. The interface defines a mapping of language codes to translation objects.

peco11 committed:
chore: add Translations interface for language support

This commit adds the Translations interface to support multiple languages in the application. The interface defines a mapping of language codes to translation objects.

peppedeka commented on src/language.ts:
```diff
@@ -0,0 +1,3 @@
+export interface Translations {
```
no ti ho detto che lo devi fare con i types e i record

peco11 committed:
chore: Update language.ts

- Add new type 'Language' with supported languages
- Change interface 'Translations' to use 'Record' type for better typing

chore: Update Translations type to allow partial language objects
